hiss/customauth/remote_backend.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth.backends import RemoteUserBackend
+from user.models import User
+from django.conf import settings
+from http.cookies import SimpleCookie
+import logging, jwt, requests
```

Comment:
```suggestion
import logging, requests
```
jwt dependancy not needed